### DISCUSSION TITLE
Shared Passwordless Auth

### DISCUSSION THREAD

**Discussion by chaseconey** [filamentphp/filament#8393] [Help]
Hey there,

I am quite new to Filament, and I trying to find the appropriate way to accomplish using shared auth (with my application) that is passwordless.

I have already set the `login()` function in the Provider to `null` like so:

```php
    public function panel(Panel $panel): Panel
    {
        return $panel
            // ...
            ->login(null)
    }
```

and added the appropriate condition in my `User` model:

```php
    public function canAccessPanel(Panel $panel): bool
    {
        return $this->is_admin;
    }
```

However, Filament still really wants to access a password on the user for some reason, but my application does not have any passwords. So I am receiving an error like so:

```
The attribute [password] either does not exist or was not retrieved for model [App\Models\User].
```

Is there some place to further override the login flow and behavior that is needed?

Any assistance would be great.

**Comment by UtkuDalmaz** [comment]
Any solution to that?

**Comment by ENRICOWWWW** [comment]
public function getPasswordAttribute(): ?string
    {
        return null;
    }
    
    
    With that at the User Model Filament wont complain anymore
